src/main/java/StartServer.java

```diff
@@ -0,0 +1,31 @@
+import server.Server;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketException;
+
+import static Validator.Validator.*;
+
+public class StartServer {
+  public static void main(String[] args) throws IOException {
+    if (validArgsLength(args) && validFlag(args)) {
+      String port = args[1];
```

Comment:
This is great for now, to parse your arguments in the main function, since you already have one. You may discover down the road that you need an argument parser that lives somewhere else, like ```argumentParser.getPortNumber()```